tests/unit/http_crawler/test_http_crawler.py

```diff
@@ -136,19 +136,48 @@ async def test_handles_redirects(
     assert server['html_endpoint'].called
 
 
+@pytest.mark.parametrize(
+    ('additional_http_error_status_codes', 'ignore_http_error_status_codes', 'expected_number_error'),
+    [
+        ([], [], 1),
+        ([403], [], 3),
+        ([], [403], 0),
+        ([403], [403], 3),
```

Comment:
So this is not considered a `SessionError`, not before this PR and not after?

Reply:
That's right